### PR TITLE
I Improved Method 1 to enable Automatic update of the user's Instance.

### PR DESCRIPTION
**Pull request by superlincoln953**
This improved method 1 will ensure that the user's self-hosted instance will be up-to-date with the latest features and bug fixes if the user set it up correctly.

### CODE REVIEW

**Review by binaryoverload — CHANGES_REQUESTED** [review]
While I agree with the sentiment, I feel like it would be better to remove the Vercel deploy button and just tell people how to set it up from scratch with a fork

This method would leave them with 2 repositories

**Comment by superlincoln953** [comment]
I removed the Vercel deploy button and told people how to set it up from scratch with a fork

File: pages/client/deploy.mdx [hunk 1]
```diff
@@ -6,11 +6,13 @@ title: 'Deploy'
 
 ## Method 1 - Vercel - Recommended
 
-[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/clone?repository-url=https%3A%2F%2Fgithub.com%2Fmovie-web%2Fmovie-web%2Ftree%2Fmaster&env=VITE_CORS_PROXY_URL,VITE_TMDB_READ_API_KEY)
-
-1. Click the Deploy button.
+1. Click [here](https://github.com/movie-web/movie-web/fork) to fork the movie-web Github repository
+1. Click [here](https://vercel.com/)to go to Vercel
```
Comment (binaryoverload) [inline pages/client/deploy.mdx:10]:
```suggestion
1. Click [here](https://vercel.com/) to go to Vercel
```

**Review by binaryoverload — APPROVED** [review]

**Comment by binaryoverload** [comment]
Great, thanks

**Pull request merged**